### PR TITLE
Iterable - update kit docs

### PR DESCRIPTION
**Pull request by julianmclain**
# Summary
- Update the Iterable Event page with documentation for the latest kit release
- Update the iOS and Android push notification pages to note that the Iterable kit supports push

### CODE REVIEW

**Comment by julianmclain** [comment]
@cla-bot check

File: src/pages/integrations/iterable/event/index.md [hunk 1]
```diff
@@ -76,29 +77,68 @@ The Iterable integration allows you to map a custom event to Iterable's [Update
...
 - [Android](https://github.com/mparticle-integrations/mparticle-android-integration-iterable)
 
-To add the Iterable Kit to your iOS or Android app, see the Cocoapods and Gradle examples below, and reference the [Apple SDK](/developers/sdk/ios/getting-started/) and [Android SDK](/developers/sdk/android/getting-started/) guides to read more about kits.
+### Adding the Kit to Your App
 
+#### iOS
+
+To install Iterable's mParticle Kit for iOS:
+- For CocoaPods, add the following dependency to your Podfile:
 :::code-selector-block
```
Comment (julianmclain) [inline src/pages/integrations/iterable/event/index.md:99]: Should `:::code-selector-block   :::` wrap each fenced code block?

File: src/pages/integrations/iterable/event/index.md [hunk 2]
```diff
@@ -76,29 +77,104 @@ The Iterable integration allows you to map a custom event to Iterable's [Update
...
+Iterable's client-side mParticle Kit enables the following features:
+- User registration with email, Customer ID or MPID
+- Push notifications
+- Rich push notifications (media and action buttons)
+- Client-side event tracking
+- Deep linking
+- In-app messages
+- Mobile Inbox
 
+The Kit automatically handles user registration, push notifications, client-side event tracking, basic in-app messaging, and deep linking by mapping mParticle’s APIs to analogous Iterable APIs. To learn more about these API mappings, you can review the source code:
```
Comment (vbabenkoru) [inline src/pages/integrations/iterable/event/index.md:89]:
```suggestion
The Kit automatically handles user registration, push notifications, basic in-app messaging, and deep linking by mapping mParticle’s APIs to analogous Iterable APIs. To learn more about these API mappings, you can review the source code:
```
Events are tracked on the server side

File: src/pages/integrations/iterable/event/index.md [hunk 3]
```diff
@@ -76,29 +77,104 @@ The Iterable integration allows you to map a custom event to Iterable's [Update
 
 ## Iterable Kit Integration
 
-mParticle also supports an Iterable Kit integration, which you can use to access Iterable's Deep Linking features. To use this integration, you need to include the Iterable kit in your installation of the mParticle SDK. The kit integration handles only Deep Linking. All other data is still sent server-to-server. The source code for each kit is available on Github:
+Iterable's client-side mParticle Kit enables the following features:
+- User registration with email, Customer ID or MPID
+- Push notifications
+- Rich push notifications (media and action buttons)
+- Client-side event tracking
```
Comment (vbabenkoru) [inline src/pages/integrations/iterable/event/index.md:84]:
```suggestion
```

**Pull request merged**